mesop/runtime/runtime.py

```diff
@@ -123,6 +124,9 @@ def register_page(self, *, path: str, page_config: PageConfig) -> None:
   def get_page_config(self, *, path: str) -> PageConfig | None:
     return self._path_to_page_config.get(path)
```

Comment:
I'm wondering if we should also add a deepcopy to this method, but that's a bit orthogonal to this PR.

Reply:
I think the page config object is small enough that the deepcopy probably wouldn't be noticeable performance-wise. And doesn't seem like it's called a lot. I can add it as a follow up.